modules/react/text-input/stories/examples/Icons.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+
+import {xSmallIcon, mailIcon} from '@workday/canvas-system-icons-web';
+import {FormField} from '@workday/canvas-kit-react/form-field';
+import {TertiaryButton} from '@workday/canvas-kit-react/button';
+import {InputGroup} from '@workday/canvas-kit-react/text-input';
+import {SystemIcon} from '@workday/canvas-kit-react/icon';
+import {dispatchInputEvent, styled} from '@workday/canvas-kit-react/common';
+
+// create a prop forwarding component for FormField to forward to
+const InputGroupFormFieldForwarder = (props: {}) => {
+  const inputRef = React.useRef(null);
+  const [value, setValue] = React.useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setValue(event.currentTarget.value);
+  };
+
+  return (
+    <InputGroup width={280}>
+      <InputGroup.Start pointerEvents="none">
+        <SystemIcon icon={mailIcon} size="small" />
+      </InputGroup.Start>
+      <InputGroup.Start pointerEvents="none">
+        <SystemIcon icon={mailIcon} size="small" />
+      </InputGroup.Start>
+      <InputGroup.Input ref={inputRef} onChange={handleChange} {...props} />
+      <InputGroup.End pointerEvents="none">
+        <SystemIcon icon={mailIcon} size="small" />
+      </InputGroup.End>
+      <InputGroup.End>
+        <TertiaryButton
+          role="presentation"
+          icon={xSmallIcon}
+          size="small"
+          tabIndex={-1}
```

Comment:
There's some context here as to why we don't want this to be focusable right?

Reply:
I added a comment